_pages/tutorials/2021-08-24-fp_error_estimation_clad.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "How to Estimate Floating Point Errors Using Automatic Differentiation"
+layout: post
+excerpt: "This tutorial demostrates how automatic differentiation can be used
+          to estimate floating point errors of algorithms."
+sitemap: false
+permalink: /tutorials/fp_error_estimation_clad_tutorial/
+date: 24-08-2021
+author: Garima Singh
+---
+
+*Tutorial level: Intro*
+
+Before we dive in, it is highly recommended that you go through the previous
+tutorials on using clad so as to make this tutorial easier to understand.
```

Comment:
do these tutorials exist somewhere? We might be specific about which ones are useful

Reply:
There are not any as of now except the clad readme.md, so that should be a to-do thing?